source-code/plugin-registry/package.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "@inlang/plugin-registry",
+	"type": "module",
+	"version": "0.0.1",
+	"private": true,
+	"exports": {
+		".": "./src/index.ts",
+		"./schema": "./src/schema.ts"
```

Comment:
1. I would export the plugin schema as part of `index.ts` thus remove the `./schema` path. 
2. I would also move plugin list to a dedicated file named `plugins.ts` that is exported by `index.ts`. Everyone can open index.ts and immediately know what is being exported and what not.

Reply:
Should be resolved